tools/walletextension/frontend/src/lib/constants.ts

```diff
@@ -1,6 +1,7 @@
-export const tenGatewayAddress = process.env.NEXT_PUBLIC_API_GATEWAY_URL || "http://127.0.0.1:3000";
+export const tenGatewayAddress =
+  process.env.NEXT_PUBLIC_API_GATEWAY_URL || "http://127.0.0.1:3000";
```

Comment:
👀  lol we should probably fix this upstream, looks like a weird hack

Reply:
lol yeah, Ziga mentioned when building the static files, it doesn't pick up the env variables. would need to investigate that.